kuma/wiki/helpers.py

```diff
@@ -318,22 +320,20 @@ def devmo_url(context, path):
     Create a URL pointing to Kuma.
     Look for a wiki page in the current locale, or default to given path
     """
+    fallback_locale = settings.WIKI_DEFAULT_LANGUAGE
+
     if hasattr(context['request'], 'locale'):
         locale = context['request'].locale
     else:
-        locale = settings.WIKI_DEFAULT_LANGUAGE
-    try:
-        url = cache.get('devmo_url:%s_%s' % (locale, path))
-    except:
-        return path
+        locale = fallback_locale
+
+    url = memcache.get('devmo_url:%s_%s' % (locale, path))
+
     if not url:
-        url = reverse('wiki.document',
-                      locale=settings.WIKI_DEFAULT_LANGUAGE,
-                      args=[path])
-        if locale != settings.WIKI_DEFAULT_LANGUAGE:
+        url = reverse('wiki.document', locale=fallback_locale, args=[path])
+        if locale != fallback_locale:
```

Comment:
If the currently selected language isn't the default language (English) then we try to get the wiki document in English because apparently the language specific page didn't exist.